docs/performance/transformers-optimization.md

```diff
@@ -0,0 +1,256 @@
+---
+title: Transformers Optimizer 
+parent: Performance
+nav_order: 4
+---
+# Transformers Optimizer
+{: .no_toc }
+
+## Contents
+{: .no_toc }
+
+* TOC placeholder
+{:toc}
+
+## Transformer Model Optimization Tool Overview
+
+ONNX Runtime automatically applies most optimizations while loading a transformer model. Some of the latest optimizations that have not yet been integrated into ONNX Runtime are available using a [separate tool](https://github.com/microsoft/onnxruntime/tree/main/onnxruntime/python/tools/transformers) that tunes models for the best performance.
+
+This tool can help in the following senarios:
+* Model is exported by tf2onnx or keras2onnx, and ONNX Runtime does not have graph optimization for them right now.
+* Convert model to use float16 to boost performance using mixed precision on GPUs with Tensor Cores (like V100 or T4).
+* Model has inputs with dynamic axis, which blocks some optimizations to be applied in ONNX Runtime due to shape inference.
+* Disable or enable some fusions to see its impact on performance or accuracy.
+
+## Installation
```

Comment:
Maybe have a workflow section ie summarize that the user first needs to convert the model then run the tool

Reply:
Or put the tool usage first and the conversion second, as this article is about the tool

Reply:
I reworked this page a bit to make it easier to follow. As of now, it was just taken from the [existing file](https://github.com/microsoft/onnxruntime/blob/main/onnxruntime/python/tools/transformers/README.md) and can be further updated later.

Reply:
Sounds good, thanks!